hooks/useDestroyer.ts

```diff
@@ -23,22 +23,30 @@ import type {
   DestroyContributionInput,
   useDestroyerDestroyContributionMutation,
 } from "@/relay/useDestroyerDestroyContributionMutation.graphql";
-
+import type {
+  DestroyOrderingInput,
+  useDestroyerDestroyOrderingMutation,
+  useDestroyerDestroyOrderingMutationResponse,
+} from "@/relay/useDestroyerDestroyOrderingMutation.graphql";
 import type { useDestroyerFragment$key } from "@/relay/useDestroyerFragment.graphql";
 import { useDestroyerFragment } from "@/relay/useDestroyerFragment.graphql";
 
+type DestroyOrdering = useDestroyerDestroyOrderingMutationResponse["destroyOrdering"];
+
 export function useDestroyer() {
   const notify = useNotify();
   const { t } = useTranslation();
 
   const handleResponse = useCallback(
-    (data: useDestroyerFragment$key | null, name: string) => {
+    (data: useDestroyerFragment$key | DestroyOrdering | null, name: string) => {
       if (!data) return;
       const results = readInlineData<useDestroyerFragment>(
         destroyFragment,
         data
       );
-      if (results.destroyed) {
+      if ("disabled" in data && data.disabled) {
+        notify.success(t("outcomes.disabled", { name }));
+      } else if (results.destroyed) {
```

Comment:
Updated types to this

Reply:
👀 This looks good to me, but @scryptmouse might have another solution. Can `disabled` be added to the `useDestroyerFragment`? @scryptmouse , will be other mutations where `disabled` is returned instead of `destroyed`?